### DISCUSSION TITLE
And... without Influx?

### DISCUSSION THREAD

**Discussion by Merca60** [jertel/vuegraf#149] [General]
Dear jertel, I know it is a "fun" request, and I'll explain.
To record datas (from a Shelly EM) I'm using an old little notebook, with a lubuntu dist, and I cannot install there InfluxDB2. I use it for its very low energy consuption, it is continuosly on, and I am doing only a test. A long test, (some months), but only a test.
I wrote a little Python code to record continuosly datas from the 2 channel Shelly to a CSV file, instead of the Json format they give with their app. 
Now I don't need professional graphs, but need to obtain datas, after, I'll search how to show them... 
I have a 19 channel Emporia Vue too, and now I ask from the app every some hour when I need seconds step to chek some change in my circuit settings.
Then, I'd like to obtain only CSV file, similar the Emporia app files, directly on my old notebook. Obviously using internet. I think it's possible, because to "send" files from Emporia Servers to Influx cloud is possible, why not to my PC?
I'm not asking to you to write this code for me, but only to confirm if my thought is or not correct.
I'm not a professional programmer, I like only to try to do things from myself, when it's possible.
Thank you, Merca60

**Comment by jertel** [comment]
Sure, you could have it write to a CSV file on the local hard disk, with some additional code. I'm curious though, why you couldn't get InfluxDB2 to run on your notebook. What are the specs of the hardware? Memory, CPU, disk, OS version?

**Reply by Merca60** [reply]
Thanks for your answer!
Then... 
PC is an ACERone with 2012 BIOS and 2GB DIMM RAM
CPU is an Intel Celeron 877; 1,4 GHz 64 bits.
It was born with Win7. About 5 years ago I installed a Linux OS, updating it when necessary.

Here some information on it, extracted from terminal.

Static hostname: AcerOne
       Icon name: computer-laptop
         Chassis: laptop
Operating System: Ubuntu 22.04.3 LTS
          Kernel: Linux 5.15.0-91-generic
    Architecture: x86-64
 Hardware Vendor: Acer
  Hardware Model: AO756
PRETTY_NAME="Ubuntu 22.04.3 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04.3 LTS (Jammy Jellyfish)"
VERSION_CODENAME=jammy
ID=ubuntu
ID_LIKE=debian
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
UBUNTU_CODENAME=jammy
Ubuntu 22.04.3 LTS \n \l

I installed the lightest GUI from the dist, I don't think it is important for "the power" of the OS.
But.. I told "cannot" because I tried to install influxDB2, to obtain a negative result from the repository. I tried to install directly too, having the same result. So I installed influx DB1, it was in my dist repository, but it "don't like" to your code...  
It's easy I made some error... and certainly I'll try again, but not immediately, next days I need to use my time for the family.

**Reply by jertel** [reply]
Influx V1 should still work, I never removed support for it. But you will need to change the version in your Vuegraf config file to 1 and use the v1 params. Ex:

```
{
    "influxDb": {
        "version": 1,
        "host": "localhost",
        "port": 8086,
        "user": "root",
        "pass": "root",
        "database": "vue"
    },
...
```

If that still doesn't work, then when you have a chance, try installing influxdb v2 again and paste the actual error message. Maybe we can figure out what's preventing it from installing.

**Reply by Merca60** [reply]
Thanks, I'll try. Be patient to have my results...
